rack-cache.gemspec

```diff
@@ -12,6 +12,7 @@ Gem::Specification.new 'rack-cache', '1.9.0' do |s|
   s.add_dependency 'rack', '>= 0.4'
 
   s.add_development_dependency 'maxitest'
+  s.add_development_dependency 'minitest', '~> 5.11.3'
```

Comment:
resolves a metric buttload of warnings regarding minitest-spec syntax changes

Reply:
`require 'maxitest/global_must'` before the autorun require should also fix it

Reply:
@grosser this caused the test failure on Ruby 2.0.x because `prepend` wasn't a thing yet, and I guess `maxitest/global_must` makes use of it...ok to switch back to locking down minitest?